projects/angular/components/ui-suggest/src/ui-suggest.component.ts

```diff
@@ -621,6 +621,10 @@ export class UiSuggestComponent extends UiSuggestMatFormFieldDirective
     @Output()
     closed = new EventEmitter<void>();
 
+    @Output()
+    // eslint-disable-next-line rxjs/finnish
+    itemSelected = new Subject<ISuggestValue>();
```

Comment:
why not event emitter?
```suggestion
    itemSelected = new EventEmitter<ISuggestValue>();
```

Reply:
Changed it